R/mwrite.R

```diff
@@ -186,6 +227,54 @@ mwrite_yaml <- function(x, file, digits = 8) {
   invisible(l)
 }
 
+#' Write a model to native mrgsolve format
+#' 
+#' Model code is written to a file in native mrgsolve format. This 
+#' can be useful for (1) breaking connection to NONMEM modeling outputs that 
+#' are imported by `$NMXML` or `$NMEXT` and (2) saving model updates (e.g., 
+#' an updated parameter list). Models can be read back using [mread()].
+#' 
+#' @inheritParams mwrite
+#' @inheritParams yaml_to_cpp
+#' 
+#' @details
+#' See important details in [mwrite_yaml()].
+#' 
+#' @return 
+#' A list containing data that was written out to the cpp file, with added 
+#' item `file`, is returned invisibly. 
+#' 
+#' @examples
+#' temp <- tempfile(fileext = ".mod")
+#' 
+#' mod <- modlib("pk1", compile = FALSE)
+#' 
+#' x <- mwrite_cpp(mod, file = temp)
+#' 
+#' mod <- mread(x$file, compile = FALSE)
+#' 
+#' mod
+#' 
+#' @seealso [mwrite_yaml()], [yaml_to_cpp()]
+#' 
+#' @md
+#' @rdname mwrite
```

Comment:
This extends mwrite.Rd in a confusing way: https://github.com/metrumresearchgroup/mrgsolve/pull/1213/files#diff-8240ad8b0e9d9eb41466c71bf50e0fbdd68e1c7036b2b152071d2107b92fb2d1

Given the fully fleshed out doc string here, I'm guessing the rdname is a leftover copy-paste error.

Reply:
Thanks; was on the fence if these should be together or not; I sort of prefer them being together b/c they share some important info, but it gets more awkward to talk about. Hopefully the crosslinking will help out.